.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "libraries/cli11/cli11"]
 	path = libraries/cli11/cli11
 	url = https://github.com/AntelopeIO/CLI11.git
+[submodule "libraries/boost"]
+	path = libraries/boost
+	url = https://github.com/AntelopeIO/boost
```

Comment:
Does this approach still require our own boost repo or does it still work with boost's repo?

Reply:
Yes it does. I'll change the url to point to `boostorg/boost`

Reply:
@spoonincode After switching to the `boostorg` repo, I have issue when creating the dev install packages because of some weird file names in boost test directories (these were not present in `AntelopeIO/boost`. I am not finding a way to exclude these from the package. Any idea?

Reply:
woah, and it was already 18 minutes in to creating the package at that point too :flushed:

In this case it seems like maybe you had a typo?
https://github.com/AntelopeIO/leap/blob/6c2040a289fe1b985bc5c3bfbbae56c0318abd5b/CMakeLists.txt#L289

Though.. trying to hoover up the whole directory like this may not be the most efficient nor canonical approach vs the cmake install rules that appear to be plumbed through. But.. the cmake install rules won't work for us as-is afaict because of the `COMPONENT` requirement, and I'm not sure how we can get over that unless we have a modification to boost's cmake files.

Reply:
You're right I had a typo! But yes it does take an incredibly long time with the boostorg repo, it was much faster with the AntelopeIO trimmed repo. 
The more I think of it, the less I'm convinced that using the boostorg repo is a good tradeof. One one hand we save an hour off from  the occasional audit (I'm happy to document the steps for comparing the AntelopeIO and boostorg repos), but then the daily use of developers and users will be more painful.

Reply:
PS: I can't believe I missed the typo. I actually looked for it but didn't see it!

Reply:
Works with the typo fixed. Thanks for noticing!